scipy/stats/stats.py

```diff
@@ -205,6 +205,9 @@
 
 
 def _chk_asarray(a, axis):
+    if np.array(a).ndim == 0:
```

Comment:
I know this is the line that @jaimefrio recommended, but it doesn't quite fit at this position. Array conversion is done twice now: here, and again in the ``asarray`` or ``ravel`` call below. It's more efficient to move the check to the end, then you can do:

    if a.ndim == 0:
        a = np.atleast_1d(a)

which is much faster.